mkt/extensions/tests/test_views.py

```diff
@@ -179,3 +179,63 @@ def test_not_an_addon(self):
                                     json.dumps({'upload': upload.pk}))
         eq_(response.status_code, 400)
         ok_(u'manifest.json' in response.json['detail'])
+
+
+class TestExtensionViewSetGet(UploadTest, RestOAuth):
+    fixtures = fixture('user_2519', 'user_999')
+
+    def setUp(self):
+        super(TestExtensionViewSetGet, self).setUp()
+        self.list_url = reverse('api-v2:extension-list')
+        self.user = UserProfile.objects.get(pk=2519)
+        self.user2 = UserProfile.objects.get(pk=999)
+        self.extension = Extension.objects.create(name=u'Mŷ Extension',
+                                                  status=STATUS_PENDING)
+        self.extension.authors.add(self.user)
+        self.extension2 = Extension.objects.create(name=u'NOT Mŷ Extension',
+                                                   status=STATUS_PENDING)
+        self.extension2.authors.add(self.user2)
+        self.url = reverse('api-v2:extension-detail',
+                           kwargs={'pk': self.extension.pk})
+        self.url2 = reverse('api-v2:extension-detail',
+                            kwargs={'pk': self.extension2.pk})
+
+    def test_list_anonymous(self):
+        response = self.anon.get(self.list_url)
+        eq_(response.status_code, 403)
+
+    def test_list_logged_in(self):
+        response = self.client.get(self.list_url)
```

Comment:
Since nowadays all our responses in tests are passed through `mkt.site.tests._JSONifiedResponse`, which does the json loading, I feel there is little boilerplate here.

Compare:

```
response = self.client.get(self.list_url)
# response.json
```

```
response, response_json = response.get()
```

The former also has the json version if needed, which is only returned if accessed, and is more explicit - it's easier to follow the test.